lstchain/data/lstchain_standard_config.json

```diff
@@ -34,6 +34,16 @@
     "leakage_intensity_width_2": [0, 1]
   },
 
+  "image_modifier": {
+    "increase_nsb": false,
+    "extra_noise_in_dim_pixels": 1.5,
+    "extra_bias_in_dim_pixels": 0.6,
+    "transition_charge": 8,
+    "extra_noise_in_bright_pixels": 1.44,
+    "increase_psf": false,
+    "smeared_light_fraction": 0.175
+  },
```

Comment:
do we really want to add this config in the standard one?

Reply:
I think it's to document the options (It is switched off by default). 

That would not be necessary if we'd directly write  a component...

Reply:
No strong opinion, it is disabled by default, but I can put it in a separate one.